mmv1/third_party/terraform/utils/provider.go.erb

```diff
@@ -381,6 +381,7 @@ products.each do |product|
 end     # products.each do
 -%>
 			},
+			// ####### START handwritten resources ###########
```

Comment:
drive-by comment: Want to delineate all 3 sections with the same formatting? That'd make it simple to code a script to count the resources / engine at a given SHA if that was desirable

Reply:
good idea, implemented